datasets/colmap.py

```diff
@@ -189,6 +189,7 @@ def setup(self, config, split):
                     img = Image.open(img_path)
                     img = img.resize(img_wh, Image.BICUBIC)
                     img = TF.to_tensor(img).permute(1, 2, 0)[...,:3]
+                    img = img.cuda() if self.config.load_data_on_gpu else img.cpu()
```

Comment:
could you please change `img.cuda()` to `img.to(self.rank)` so that this works on DDP?